apps/frontend/src/app/Data/Characters/KujouSara/index.tsx

```diff
@@ -284,34 +296,28 @@ const sheet: ICharacterSheet = {
           },
           {
             text: ct.chg('burst.skillParams.2'),
-            value: `${dm.burst.cd}s`,
+            value: dm.burst.cd,
+            unit: 's',
           },
           {
             text: ct.chg('burst.skillParams.3'),
-            value: `${dm.burst.enerCost}`,
+            value: dm.burst.enerCost,
+            unit: 's',
           },
         ],
       },
     ]),
 
     passive1: ct.talentTem('passive1'),
-    passive2: ct.talentTem('passive2', [
-      ct.fieldsTem('passive2', {
-        fields: [
-          {
-            node: dmgFormulas.passive2.energyRegen,
-          },
-        ],
-      }),
-    ]),
+    passive2: ct.talentTem('passive2'),
     passive3: ct.talentTem('passive3'),
     constellation1: ct.talentTem('constellation1'),
     constellation2: ct.talentTem('constellation2', [
       ct.fieldsTem('constellation2', {
         fields: [
           {
             node: infoMut(dmgFormulas.constellation2.dmg, {
-              name: ct.chg(`skill.skillParams.0`),
+              name: st('dmg'),
```

Comment:
That's an issue I've noticed with a few conditionals and these damage instances. It feels like it isn't perfect to use the generic strings (like `afterUse.burst` or `dmg`), but also like it's undeserving of a custom name. Maybe I can come back to this later? I feel like it's fine to keep it this way for now.

By the way, I think I mentioned this before, I changed the name because the old one was the Skill's damage name. And they aren't the same.